### PR TITLE
feat: image component

### PR DESCRIPTION
**Pull request by way2nizam**
1. responsive image
2. round image with different sizes
3. responsive round image
4. responsive round image with border

### CODE REVIEW

File: image-component/image-component.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.img--large {
+  max-width: 8rem;
+}
+
+.img--round {
+  border-radius: 50%;
+}
+
+.img--bordered {
+  border: 1.5rem solid goldenrod;
```
Comment (parthgabani777) [inline image-component/image-component.css]: try to use CSS variable for color

Comment (way2nizam) [reply image-component/image-component.css]: Okay noted

**Review by ishapatell1 — APPROVED** [review]
The image component Looks good enough to merge.

**Review by parthgabani777 — APPROVED** [review]
looks good. you can merge it.

**Pull request merged**